kolibri/core/content/tasks.py

```diff
@@ -335,6 +336,18 @@ def remoteresourceimport(
     import_manager.run()
 
 
+@register_task(
+    queue=QUEUE,
+    long_running=False,
```

Comment:
It might be safer to set this to `True` - I am still a little iffy about how exactly we should be determining this, but I tend to assume that importing resources might always take > 10 minutes on some occasions.